src/tim/vx/ops/stridedslice_test.cc

```diff
@@ -23,7 +23,8 @@
 *****************************************************************************/
 #include "tim/vx/context.h"
 #include "tim/vx/graph.h"
-#include "tim/vx/ops/stridedslice.h"
+#include "tim/vx/ops.h"
+#include "tim/transform/layout_inference.h"
```

Comment:
move layout related test to layout test. Here, we only put op test without layout inference.